src/dom/element.ts

```diff
@@ -801,6 +801,18 @@ export class Element extends Node {
     }
   }
 
+  toggleAttribute(rawName: string, force?: boolean) {
+    const name = String(rawName?.toLowerCase());
+    if (force === undefined) {
+      force = !this.hasAttribute(name);
+    }
+    if (force) {
+      this.setAttribute(name, "");
+    } else {
+      this.removeAttribute(name);
+    }
+  }
```

Comment:
This implementation doesn't follow the [standard behaviour](https://dom.spec.whatwg.org/#ref-for-dom-element-toggleattribute%E2%91%A0) of returning a boolean value and also not overwriting the value if it's already set when `force === true`. Generally, I look at the spec when implementing things to follow it loosely, for example in:

![image](https://github.com/b-fuze/deno-dom/assets/10504957/c530a035-f942-43d4-aa41-0e1b5d6d7f48)

I will mostly ignore step 1. Yes, that's incorrect, but for deno-dom's purposes I don't think that it's super important. You're already doing step 2. Step 3 doesn't fully apply to deno-dom because we don't have an attribute list, but the idea in general applies. The rest of the steps all apply, implemented generally how you've done here is fine just as long as the important bits (attribute setting behaviour, return value) are satisfied

Reply:
It should be more compliant with the spec now

I missed the part about the return value since I only looked at the mdn doc rather than the dom spec (I'll look into it for future prs)